packages/flutter_tools/test/src/context.dart

```diff
@@ -254,6 +254,11 @@ class FakeDeviceManager implements DeviceManager {
   }) {
     return TestDeviceDiscoverySupportFilter();
   }
+
+  @override
+  List<Device> prioritizeEphemeralDevices(List<Device> devices) {
+    return devices;
+  }
```

Comment:
I've noticed a few places in your code in previous PRs where you could have used `=>`, not sure if you're aware of that arrow dart syntax (this isn't even a nit comment, you don't have to change this).